Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BoxViewportEdit.cpp

```diff
@@ -92,29 +95,35 @@ namespace AzToolsFramework
                     [this, entityComponentIdPair](
                         const LinearManipulator::Action& action)
                 {
+                    AZ::Transform boxLocalTransform = AZ::Transform::CreateIdentity();
+                    BoxManipulatorRequestBus::EventResult(
+                        boxLocalTransform, entityComponentIdPair, &BoxManipulatorRequests::GetCurrentLocalTransform);
```

Comment:
question: I take it `GetCurrentLocalTransform` will be the identity for the `BoxShape` and a unique value for the `BoxCollider`?

Reply:
Yeah, although that might change if we make the changes we've previously discussed to allow box shapes to have offsets / independently adjustable ends.

Reply:
I see understood 👍